apps/processing/.env.example

```diff
@@ -13,5 +13,7 @@ INDEXER_ADMIN_SECRET=testing
 
 IPFS_GATEWAYS_URL=["https://ipfs.io","https://gateway.pinata.cloud","https://dweb.link", "https://ipfs.eth.aragon.network"]
 
+PRICING_SOURCE= #coingecko | dummy
```

Comment:
Is that `#coingecko` ok?

Reply:
is # for comment, in next PR i'll add a whitespace